### ISSUE TITLE
<html> element does not have a valid value for its [lang] attribute.

### ISSUE THREAD

**Issue by Milos2504** [tobimori/kirby-seo#77]
When using <?= $site->lang() ?> as the lang attribute in the <html> tag, Chrome Lighthouse flags it as an issue under Accessibility "<html> element does not have a valid value for its [lang] attribute.". The returned value of the $site->lang() is 'en_US' while it needs to be 'en-US'.
